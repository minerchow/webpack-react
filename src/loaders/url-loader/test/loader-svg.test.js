import webpack from './helpers/compiler';

describe('Loader', () => {
  it('should works', async () => {
    const config = {
      loader: {
        test: /\.svg$/,
        options: {},
      },
    };

    const stats = await webpack('fixture-svg.js', config);
    const [{ source }] = stats.toJson().modules;

    expect(source).toMatchSnapshot();
  });
});
