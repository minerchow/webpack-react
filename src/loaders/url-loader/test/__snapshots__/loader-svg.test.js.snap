// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should works 1`] = `"module.exports = \\"data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+Cjxzdmcgd2lkdGg9Ijc1cHgiIGhlaWdodD0iNzVweCIgdmlld0JveD0iMCAwIDc1IDc1IiB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiPgogICAgPCEtLSBHZW5lcmF0b3I6IFNrZXRjaCAzLjcuMSAoMjgyMTUpIC0gaHR0cDovL3d3dy5ib2hlbWlhbmNvZGluZy5jb20vc2tldGNoIC0tPgogICAgPHRpdGxlPm1vY2hpPC90aXRsZT4KICAgIDxkZXNjPkNyZWF0ZWQgd2l0aCBTa2V0Y2guPC9kZXNjPgogICAgPGRlZnM+CiAgICAgICAgPHJlY3QgaWQ9InBhdGgtMSIgeD0iMCIgeT0iMCIgd2lkdGg9IjcwIiBoZWlnaHQ9IjcwIiByeD0iMyI+PC9yZWN0PgogICAgICAgIDxtYXNrIGlkPSJtYXNrLTIiIG1hc2tDb250ZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIiBtYXNrVW5pdHM9Im9iamVjdEJvdW5kaW5nQm94IiB4PSIwIiB5PSIwIiB3aWR0aD0iNzAiIGhlaWdodD0iNzAiIGZpbGw9IndoaXRlIj4KICAgICAgICAgICAgPHVzZSB4bGluazpocmVmPSIjcGF0aC0xIj48L3VzZT4KICAgICAgICA8L21hc2s+CiAgICA8L2RlZnM+CiAgICA8ZyBpZD0iUGFnZS0xIiBzdHJva2U9Im5vbmUiIHN0cm9rZS13aWR0aD0iMSIgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj4KICAgICAgICA8ZyBpZD0ibW9jaGkiPgogICAgICAgICAgICA8ZyBpZD0iR3JvdXAtMTUiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDEuMDAwMDAwLCAxLjAwMDAwMCkiPgogICAgICAgICAgICAgICAgPHJlY3QgaWQ9IlJlY3RhbmdsZS02OC1Db3B5LTciIGZpbGw9IiM0MTQxNDEiIG9wYWNpdHk9IjAuMTQ5MDU1NTA0IiB4PSIzIiB5PSIzIiB3aWR0aD0iNzAiIGhlaWdodD0iNzAiIHJ4PSIzIj48L3JlY3Q+CiAgICAgICAgICAgICAgICA8dXNlIGlkPSJSZWN0YW5nbGUtNjgiIHN0cm9rZT0iIzYzNjM2MyIgbWFzaz0idXJsKCNtYXNrLTIpIiBzdHJva2Utd2lkdGg9IjUiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgZmlsbD0iI0ZGRkZGRiIgeGxpbms6aHJlZj0iI3BhdGgtMSI+PC91c2U+CiAgICAgICAgICAgICAgICA8ZyBpZD0iR3JvdXAtOSIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoNy4wMDAwMDAsIDguMDAwMDAwKSI+CiAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTEyLjcwODc5NTUsMi43NjE2Njc4NiBMNDEuNzA4Nzk1NSwyLjc2MTY2Nzg2IEwxMi43MDg3OTU1LDIuNzYxNjY3ODYgWiIgaWQ9IlBhdGgtNDEiIHN0cm9rZT0iIzYzNjM2MyIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgyNy4yMDg3OTUsIDIuNzYxNjY4KSBzY2FsZSgtMSwgMSkgdHJhbnNsYXRlKC0yNy4yMDg3OTUsIC0yLjc2MTY2OCkgIj48L3BhdGg+CiAgICAgICAgICAgICAgICAgICAgPGcgaWQ9Ikdyb3VwLTciPgogICAgICAgICAgICAgICAgICAgICAgICA8cGF0aCBkPSJNMzEuMzM1OTE0Miw1NS41NjI5MjU4IEM0NC45NzIzNTc0LDU1LjU2MjkyNTggNTYuMDI2ODc4OSw0NC41MDg0MDQ0IDU2LjAyNjg3ODksMzAuODcxOTYxMSBDNTYuMDI2ODc4OSwxNy4yMzU1MTc4IDQ0Ljk3MjM1NzQsNi4xODA5OTY0MiAzMS4zMzU5MTQyLDYuMTgwOTk2NDIgQzE3LjY5OTQ3MDksNi4xODA5OTY0MiA2LjY0NDk0OTUsMTcuMjM1NTE3OCA2LjY0NDk0OTUsMzAuODcxOTYxMSBDNi42NDQ5NDk1LDQ0LjUwODQwNDQgMTcuNjk5NDcwOSw1NS41NjI5MjU4IDMxLjMzNTkxNDIsNTUuNTYyOTI1OCBaIiBpZD0iT3ZhbC0zMS1Db3B5LTUiIGZpbGw9IiM0MTQxNDEiIG9wYWNpdHk9IjAuMTQ5MDU1NTA0Ij48L3BhdGg+CiAgICAgICAgICAgICAgICAgICAgICAgIDxwYXRoIGQ9Ik0yOC4zMzU5MTQyLDUyLjU2MjkyNTggQzQxLjk3MjM1NzQsNTIuNTYyOTI1OCA1My4wMjY4Nzg5LDQxLjUwODQwNDQgNTMuMDI2ODc4OSwyNy44NzE5NjExIEM1My4wMjY4Nzg5LDE0LjIzNTUxNzggNDEuOTcyMzU3NCwzLjE4MDk5NjQyIDI4LjMzNTkxNDIsMy4xODA5OTY0MiBDMTQuNjk5NDcwOSwzLjE4MDk5NjQyIDMuNjQ0OTQ5NSwxNC4yMzU1MTc4IDMuNjQ0OTQ5NSwyNy44NzE5NjExIEMzLjY0NDk0OTUsNDEuNTA4NDA0NCAxNC42OTk0NzA5LDUyLjU2MjkyNTggMjguMzM1OTE0Miw1Mi41NjI5MjU4IFoiIGlkPSJPdmFsLTMxIiBzdHJva2U9IiM2MzYzNjMiIHN0cm9rZS13aWR0aD0iMi41IiBmaWxsPSIjRkZGRkZGIj48L3BhdGg+CiAgICAgICAgICAgICAgICAgICAgICAgIDxwYXRoIGQ9Ik0yMy4wNzk0OTI1LDcuMDQ3NTg5OTcgQzE2LjcwNzEyMjUsOC43NDQ5MTU3OSAxMS42NDU0NTI1LDEyLjk3MDQwMiA4Ljk4OTgxMjM4LDE4LjQ5OTkyMTkiIGlkPSJQYXRoLTIxIiBzdHJva2U9IiM2MzYzNjMiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCI+PC9wYXRoPgogICAgICAgICAgICAgICAgICAgICAgICA8ZWxsaXBzZSBpZD0iT3ZhbC0zMiIgZmlsbD0iI0YzQTVBNSIgY3g9IjQwLjgyMTExODQiIGN5PSIyOC42NjI1MDIxIiByeD0iMy41NzU1OTk1OCIgcnk9IjIuMzEyMjY2MjIiPjwvZWxsaXBzZT4KICAgICAgICAgICAgICAgICAgICAgICAgPGVsbGlwc2UgaWQ9Ik92YWwtMzIiIGZpbGw9IiNGM0E1QTUiIGN4PSIxNy41MzE0MDIyIiBjeT0iMjguNDk0MDI5OCIgcng9IjMuNTc1NTk5NTgiIHJ5PSIyLjMxMjI2NjIyIj48L2VsbGlwc2U+CiAgICAgICAgICAgICAgICAgICAgICAgIDxwYXRoIGQ9Ik0zOS4zNzA2NTcyLDI3IEw2OC4zNzA2NTcyLDI3IEwzOS4zNzA2NTcyLDI3IFoiIGlkPSJQYXRoLTQxIiBzdHJva2U9IiM2MzYzNjMiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoNTMuODcwNjU3LCAyNy4wMDAwMDApIHJvdGF0ZSgtMjcwLjAwMDAwMCkgdHJhbnNsYXRlKC01My44NzA2NTcsIC0yNy4wMDAwMDApICI+PC9wYXRoPgogICAgICAgICAgICAgICAgICAgICAgICA8cGF0aCBkPSJNNTIuNDE2MTExNywyNi44ODI4MzEgQzUzLjE0MzM4NDQsMjYuMTU1NTU4MyA1NC41OTc5Mjk5LDI2LjE1NTU1ODMgNTUuMzI1MjAyNiwyNi44ODI4MzEgQzU2LjA1MjQ3NTMsMjcuNjEwMTAzNyA1Ni4wNTI0NzUzLDI5LjA2NDY0OTIgNTUuMzI1MjAyNiwyOS43OTE5MjE5IEM1NC41OTc5Mjk5LDMwLjUxOTE5NDYgNTMuMTQzMzg0NCwzMC41MTkxOTQ2IDUyLjQxNjExMTcsMjkuNzkxOTIxOSBDNTEuNjg4ODM5LDI5LjA2NDY0OTIgNTEuNjg4ODM5LDI3LjYxMDEwMzcgNTIuNDE2MTExNywyNi44ODI4MzEgWiIgaWQ9IlJlY3RhbmdsZS0zMCIgc3Ryb2tlPSIjNjM2MzYzIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIGZpbGw9IiNGRkZGRkYiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDUzLjg3MDY1NywgMjguMzM3Mzc2KSByb3RhdGUoLTI3MC4wMDAwMDApIHRyYW5zbGF0ZSgtNTMuODcwNjU3LCAtMjguMzM3Mzc2KSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTUyLjQxNjExMTcsMTEuODgyODMxIEM1My4xNDMzODQ0LDExLjE1NTU1ODMgNTQuNTk3OTI5OSwxMS4xNTU1NTgzIDU1LjMyNTIwMjYsMTEuODgyODMxIEM1Ni4wNTI0NzUzLDEyLjYxMDEwMzcgNTYuMDUyNDc1MywxNC4wNjQ2NDkyIDU1LjMyNTIwMjYsMTQuNzkxOTIxOSBDNTQuNTk3OTI5OSwxNS41MTkxOTQ2IDUzLjE0MzM4NDQsMTUuNTE5MTk0NiA1Mi40MTYxMTE3LDE0Ljc5MTkyMTkgQzUxLjY4ODgzOSwxNC4wNjQ2NDkyIDUxLjY4ODgzOSwxMi42MTAxMDM3IDUyLjQxNjExMTcsMTEuODgyODMxIFoiIGlkPSJSZWN0YW5nbGUtMzAiIHN0cm9rZT0iIzYzNjM2MyIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIiBmaWxsPSIjRkZGRkZGIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSg1My44NzA2NTcsIDEzLjMzNzM3Nikgcm90YXRlKC0yNzAuMDAwMDAwKSB0cmFuc2xhdGUoLTUzLjg3MDY1NywgLTEzLjMzNzM3NikgIj48L3BhdGg+CiAgICAgICAgICAgICAgICAgICAgICAgIDxwYXRoIGQ9Ik01Mi40MTYxMTE3LDQxLjg4MjgzMSBDNTMuMTQzMzg0NCw0MS4xNTU1NTgzIDU0LjU5NzkyOTksNDEuMTU1NTU4MyA1NS4zMjUyMDI2LDQxLjg4MjgzMSBDNTYuMDUyNDc1Myw0Mi42MTAxMDM3IDU2LjA1MjQ3NTMsNDQuMDY0NjQ5MiA1NS4zMjUyMDI2LDQ0Ljc5MTkyMTkgQzU0LjU5NzkyOTksNDUuNTE5MTk0NiA1My4xNDMzODQ0LDQ1LjUxOTE5NDYgNTIuNDE2MTExNyw0NC43OTE5MjE5IEM1MS42ODg4MzksNDQuMDY0NjQ5MiA1MS42ODg4MzksNDIuNjEwMTAzNyA1Mi40MTYxMTE3LDQxLjg4MjgzMSBaIiBpZD0iUmVjdGFuZ2xlLTMwIiBzdHJva2U9IiM2MzYzNjMiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgZmlsbD0iI0ZGRkZGRiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoNTMuODcwNjU3LCA0My4zMzczNzYpIHJvdGF0ZSgtMjcwLjAwMDAwMCkgdHJhbnNsYXRlKC01My44NzA2NTcsIC00My4zMzczNzYpICI+PC9wYXRoPgogICAgICAgICAgICAgICAgICAgICAgICA8cGF0aCBkPSJNLTExLjYyOTM0MjgsMjcgTDE3LjM3MDY1NzIsMjcgTC0xMS42MjkzNDI4LDI3IFoiIGlkPSJQYXRoLTQxIiBzdHJva2U9IiM2MzYzNjMiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMi44NzA2NTcsIDI3LjAwMDAwMCkgcm90YXRlKC0yNzAuMDAwMDAwKSB0cmFuc2xhdGUoLTIuODcwNjU3LCAtMjcuMDAwMDAwKSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTEuNDE2MTExNywyNi44ODI4MzEgQzIuMTQzMzg0NDIsMjYuMTU1NTU4MyAzLjU5NzkyOTg4LDI2LjE1NTU1ODMgNC4zMjUyMDI2MSwyNi44ODI4MzEgQzUuMDUyNDc1MzMsMjcuNjEwMTAzNyA1LjA1MjQ3NTMzLDI5LjA2NDY0OTIgNC4zMjUyMDI2MSwyOS43OTE5MjE5IEMzLjU5NzkyOTg4LDMwLjUxOTE5NDYgMi4xNDMzODQ0MiwzMC41MTkxOTQ2IDEuNDE2MTExNywyOS43OTE5MjE5IEMwLjY4ODgzODk2OSwyOS4wNjQ2NDkyIDAuNjg4ODM4OTY5LDI3LjYxMDEwMzcgMS40MTYxMTE3LDI2Ljg4MjgzMSBaIiBpZD0iUmVjdGFuZ2xlLTMwIiBzdHJva2U9IiM2MzYzNjMiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgZmlsbD0iI0ZGRkZGRiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMi44NzA2NTcsIDI4LjMzNzM3Nikgcm90YXRlKC0yNzAuMDAwMDAwKSB0cmFuc2xhdGUoLTIuODcwNjU3LCAtMjguMzM3Mzc2KSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTEuNDE2MTExNywxMS44ODI4MzEgQzIuMTQzMzg0NDIsMTEuMTU1NTU4MyAzLjU5NzkyOTg4LDExLjE1NTU1ODMgNC4zMjUyMDI2MSwxMS44ODI4MzEgQzUuMDUyNDc1MzMsMTIuNjEwMTAzNyA1LjA1MjQ3NTMzLDE0LjA2NDY0OTIgNC4zMjUyMDI2MSwxNC43OTE5MjE5IEMzLjU5NzkyOTg4LDE1LjUxOTE5NDYgMi4xNDMzODQ0MiwxNS41MTkxOTQ2IDEuNDE2MTExNywxNC43OTE5MjE5IEMwLjY4ODgzODk2OSwxNC4wNjQ2NDkyIDAuNjg4ODM4OTY5LDEyLjYxMDEwMzcgMS40MTYxMTE3LDExLjg4MjgzMSBaIiBpZD0iUmVjdGFuZ2xlLTMwIiBzdHJva2U9IiM2MzYzNjMiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgZmlsbD0iI0ZGRkZGRiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMi44NzA2NTcsIDEzLjMzNzM3Nikgcm90YXRlKC0yNzAuMDAwMDAwKSB0cmFuc2xhdGUoLTIuODcwNjU3LCAtMTMuMzM3Mzc2KSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTEuNDE2MTExNyw0MS44ODI4MzEgQzIuMTQzMzg0NDIsNDEuMTU1NTU4MyAzLjU5NzkyOTg4LDQxLjE1NTU1ODMgNC4zMjUyMDI2MSw0MS44ODI4MzEgQzUuMDUyNDc1MzMsNDIuNjEwMTAzNyA1LjA1MjQ3NTMzLDQ0LjA2NDY0OTIgNC4zMjUyMDI2MSw0NC43OTE5MjE5IEMzLjU5NzkyOTg4LDQ1LjUxOTE5NDYgMi4xNDMzODQ0Miw0NS41MTkxOTQ2IDEuNDE2MTExNyw0NC43OTE5MjE5IEMwLjY4ODgzODk2OSw0NC4wNjQ2NDkyIDAuNjg4ODM4OTY5LDQyLjYxMDEwMzcgMS40MTYxMTE3LDQxLjg4MjgzMSBaIiBpZD0iUmVjdGFuZ2xlLTMwIiBzdHJva2U9IiM2MzYzNjMiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgZmlsbD0iI0ZGRkZGRiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMi44NzA2NTcsIDQzLjMzNzM3Nikgcm90YXRlKC0yNzAuMDAwMDAwKSB0cmFuc2xhdGUoLTIuODcwNjU3LCAtNDMuMzM3Mzc2KSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTEyLjcwODc5NTUsNTIuNzYxNjY3OSBMNDEuNzA4Nzk1NSw1Mi43NjE2Njc5IEwxMi43MDg3OTU1LDUyLjc2MTY2NzkgWiIgaWQ9IlBhdGgtNDEiIHN0cm9rZT0iIzYzNjM2MyIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgyNy4yMDg3OTUsIDUyLjc2MTY2OCkgc2NhbGUoLTEsIDEpIHRyYW5zbGF0ZSgtMjcuMjA4Nzk1LCAtNTIuNzYxNjY4KSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTI2Ljc5NDYzMyw1MS4zMDcxMjI0IEMyNy41MjE5MDU3LDUwLjU3OTg0OTcgMjguOTc2NDUxMSw1MC41Nzk4NDk3IDI5LjcwMzcyMzksNTEuMzA3MTIyNCBDMzAuNDMwOTk2Niw1Mi4wMzQzOTUxIDMwLjQzMDk5NjYsNTMuNDg4OTQwNiAyOS43MDM3MjM5LDU0LjIxNjIxMzMgQzI4Ljk3NjQ1MTEsNTQuOTQzNDg2IDI3LjUyMTkwNTcsNTQuOTQzNDg2IDI2Ljc5NDYzMyw1NC4yMTYyMTMzIEMyNi4wNjczNjAyLDUzLjQ4ODk0MDYgMjYuMDY3MzYwMiw1Mi4wMzQzOTUxIDI2Ljc5NDYzMyw1MS4zMDcxMjI0IFoiIGlkPSJSZWN0YW5nbGUtMzAiIHN0cm9rZT0iIzYzNjM2MyIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIiBmaWxsPSIjRkZGRkZGIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgyOC4yNDkxNzgsIDUyLjc2MTY2OCkgc2NhbGUoLTEsIDEpIHRyYW5zbGF0ZSgtMjguMjQ5MTc4LCAtNTIuNzYxNjY4KSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTQxLjc5Njc1LDUxLjMwNzEyMjQgQzQyLjUyNDAyMjgsNTAuNTc5ODQ5NyA0My45Nzg1NjgyLDUwLjU3OTg0OTcgNDQuNzA1ODQwOSw1MS4zMDcxMjI0IEM0NS40MzMxMTM3LDUyLjAzNDM5NTEgNDUuNDMzMTEzNyw1My40ODg5NDA2IDQ0LjcwNTg0MDksNTQuMjE2MjEzMyBDNDMuOTc4NTY4Miw1NC45NDM0ODYgNDIuNTI0MDIyOCw1NC45NDM0ODYgNDEuNzk2NzUsNTQuMjE2MjEzMyBDNDEuMDY5NDc3Myw1My40ODg5NDA2IDQxLjA2OTQ3NzMsNTIuMDM0Mzk1MSA0MS43OTY3NSw1MS4zMDcxMjI0IFoiIGlkPSJSZWN0YW5nbGUtMzAiIHN0cm9rZT0iIzYzNjM2MyIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIiBmaWxsPSIjRkZGRkZGIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSg0My4yNTEyOTUsIDUyLjc2MTY2OCkgc2NhbGUoLTEsIDEpIHRyYW5zbGF0ZSgtNDMuMjUxMjk1LCAtNTIuNzYxNjY4KSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTEwLjc5Njc1LDUxLjMwNzEyMjQgQzExLjUyNDAyMjgsNTAuNTc5ODQ5NyAxMi45Nzg1NjgyLDUwLjU3OTg0OTcgMTMuNzA1ODQwOSw1MS4zMDcxMjI0IEMxNC40MzMxMTM3LDUyLjAzNDM5NTEgMTQuNDMzMTEzNyw1My40ODg5NDA2IDEzLjcwNTg0MDksNTQuMjE2MjEzMyBDMTIuOTc4NTY4Miw1NC45NDM0ODYgMTEuNTI0MDIyOCw1NC45NDM0ODYgMTAuNzk2NzUsNTQuMjE2MjEzMyBDMTAuMDY5NDc3Myw1My40ODg5NDA2IDEwLjA2OTQ3NzMsNTIuMDM0Mzk1MSAxMC43OTY3NSw1MS4zMDcxMjI0IFoiIGlkPSJSZWN0YW5nbGUtMzAiIHN0cm9rZT0iIzYzNjM2MyIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIiBmaWxsPSIjRkZGRkZGIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxMi4yNTEyOTUsIDUyLjc2MTY2OCkgc2NhbGUoLTEsIDEpIHRyYW5zbGF0ZSgtMTIuMjUxMjk1LCAtNTIuNzYxNjY4KSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTI2Ljc5NDYzMywxLjMwNzEyMjQxIEMyNy41MjE5MDU3LDAuNTc5ODQ5Njc4IDI4Ljk3NjQ1MTEsMC41Nzk4NDk2NzggMjkuNzAzNzIzOSwxLjMwNzEyMjQxIEMzMC40MzA5OTY2LDIuMDM0Mzk1MTMgMzAuNDMwOTk2NiwzLjQ4ODk0MDU5IDI5LjcwMzcyMzksNC4yMTYyMTMzMSBDMjguOTc2NDUxMSw0Ljk0MzQ4NjA0IDI3LjUyMTkwNTcsNC45NDM0ODYwNCAyNi43OTQ2MzMsNC4yMTYyMTMzMSBDMjYuMDY3MzYwMiwzLjQ4ODk0MDU5IDI2LjA2NzM2MDIsMi4wMzQzOTUxMyAyNi43OTQ2MzMsMS4zMDcxMjI0MSBaIiBpZD0iUmVjdGFuZ2xlLTMwIiBzdHJva2U9IiM2MzYzNjMiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgZmlsbD0iI0ZGRkZGRiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMjguMjQ5MTc4LCAyLjc2MTY2OCkgc2NhbGUoLTEsIDEpIHRyYW5zbGF0ZSgtMjguMjQ5MTc4LCAtMi43NjE2NjgpICI+PC9wYXRoPgogICAgICAgICAgICAgICAgICAgICAgICA8cGF0aCBkPSJNNDEuNzk2NzUsMS4zMDcxMjI0MSBDNDIuNTI0MDIyOCwwLjU3OTg0OTY3OCA0My45Nzg1NjgyLDAuNTc5ODQ5Njc4IDQ0LjcwNTg0MDksMS4zMDcxMjI0MSBDNDUuNDMzMTEzNywyLjAzNDM5NTEzIDQ1LjQzMzExMzcsMy40ODg5NDA1OSA0NC43MDU4NDA5LDQuMjE2MjEzMzEgQzQzLjk3ODU2ODIsNC45NDM0ODYwNCA0Mi41MjQwMjI4LDQuOTQzNDg2MDQgNDEuNzk2NzUsNC4yMTYyMTMzMSBDNDEuMDY5NDc3MywzLjQ4ODk0MDU5IDQxLjA2OTQ3NzMsMi4wMzQzOTUxMyA0MS43OTY3NSwxLjMwNzEyMjQxIFoiIGlkPSJSZWN0YW5nbGUtMzAiIHN0cm9rZT0iIzYzNjM2MyIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIiBmaWxsPSIjRkZGRkZGIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSg0My4yNTEyOTUsIDIuNzYxNjY4KSBzY2FsZSgtMSwgMSkgdHJhbnNsYXRlKC00My4yNTEyOTUsIC0yLjc2MTY2OCkgIj48L3BhdGg+CiAgICAgICAgICAgICAgICAgICAgICAgIDxwYXRoIGQ9Ik0xMC43OTY3NSwxLjMwNzEyMjQxIEMxMS41MjQwMjI4LDAuNTc5ODQ5Njc4IDEyLjk3ODU2ODIsMC41Nzk4NDk2NzggMTMuNzA1ODQwOSwxLjMwNzEyMjQxIEMxNC40MzMxMTM3LDIuMDM0Mzk1MTMgMTQuNDMzMTEzNywzLjQ4ODk0MDU5IDEzLjcwNTg0MDksNC4yMTYyMTMzMSBDMTIuOTc4NTY4Miw0Ljk0MzQ4NjA0IDExLjUyNDAyMjgsNC45NDM0ODYwNCAxMC43OTY3NSw0LjIxNjIxMzMxIEMxMC4wNjk0NzczLDMuNDg4OTQwNTkgMTAuMDY5NDc3MywyLjAzNDM5NTEzIDEwLjc5Njc1LDEuMzA3MTIyNDEgWiIgaWQ9IlJlY3RhbmdsZS0zMCIgc3Ryb2tlPSIjNjM2MzYzIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIGZpbGw9IiNGRkZGRkYiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDEyLjI1MTI5NSwgMi43NjE2NjgpIHNjYWxlKC0xLCAxKSB0cmFuc2xhdGUoLTEyLjI1MTI5NSwgLTIuNzYxNjY4KSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTI2Ljg1NzMwNTIsMzEuOTAzMjIwMyBDMjYuODU3MzA1MiwzMC44MjQ0MDI5IDI3LjcxNTM2MiwyOS43NTM3NTI5IDI5LjE2NDEzMDIsMjkuNzUzNzUzIEMzMC42MTI4OTg1LDI5Ljc1Mzc1MyAzMS40OTUyMTU0LDMwLjg2NDg3NjggMzEuNDk1MjE1NCwzMS45ODQ1OTE0IiBpZD0iUGF0aC0xODUtQ29weSIgc3Ryb2tlPSIjNjM2MzYzIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDI5LjE3NjI2MCwgMzAuODY5MTcyKSByb3RhdGUoLTE4MC4wMDAwMDApIHRyYW5zbGF0ZSgtMjkuMTc2MjYwLCAtMzAuODY5MTcyKSAiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTE3Ljc0MDg2NDgsMjUuMDc2MDA2MyBDMTcuNzQwODY0OCwyNS4wNzYwMDYzIDE3LjQ3OTk5MDcsMjEuNTM2ODIxOSAyMC4yMzY0MDg2LDIxLjUzNjgyMjEgQzIyLjc4MDgxOCwyMS41MzY4MjIzIDIyLjczMTk1MjMsMjUuMDc2MDA2MyAyMi43MzE5NTIzLDI1LjA3NjAwNjMiIGlkPSJQYXRoLTcxLUNvcHktMiIgc3Ryb2tlPSIjNjM2MzYzIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTM1LjAwNTQxNDEsMjUuMDc2MDA2MyBDMzUuMDA1NDE0MSwyNS4wNzYwMDYzIDM0Ljc0NDU0LDIxLjUzNjgyMTkgMzcuNTAwOTU3OCwyMS41MzY4MjIxIEM0MC4wNDUzNjcyLDIxLjUzNjgyMjMgMzkuOTk2NTAxNSwyNS4wNzYwMDYzIDM5Ljk5NjUwMTUsMjUuMDc2MDA2MyIgaWQ9IlBhdGgtNzEtQ29weSIgc3Ryb2tlPSIjNjM2MzYzIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICA8L2c+CiAgICAgICAgICAgICAgICA8L2c+CiAgICAgICAgICAgIDwvZz4KICAgICAgICA8L2c+CiAgICA8L2c+Cjwvc3ZnPg==\\""`;
